src/plugins/gui/providers/paybisProvider.ts

```diff
@@ -761,6 +761,8 @@ export const paybisProvider: FiatProviderFactory = {
                       await openWebView()
                       if (e.message === SendErrorNoTransaction) {
                         await showUi.showToast(lstrings.fiat_plugin_sell_failed_to_send_try_again, NOT_SUCCESS_TOAST_HIDE_MS)
+                      } else if (e.message === SendErrorBackPressed) {
```

Comment:
This should also be `catch (e: unknown) {` above, in which case we need to add the `e instanceof Error` bits too.